package.json

```diff
@@ -43,6 +43,7 @@
   "dependencies": {
     "@hapi/cryptiles": "5.0.0",
     "@hapi/wreck": "^17.1.0",
+    "@opensearch-project/oui": "^1.6.0",
```

Comment:
Added this dependency in the new component. However, this dependency has added a few run-time warnings. 
<details>
<summary>Expand to see one of the newly introduced warnings: </summary>

```cli
WARNING in ./node_modules/@opensearch-project/oui/lib/eui_components/icon/icon.js 649:39-49
       │          Critical dependency: the request of a dependency is an expression
       │              at CommonJsRequireContextDependency.getWarnings (/Users/dchanp/Documents/amazon/codebase/OpenSearch-Dashboards/node_modules/webpack/lib/dependencies/ContextDependency.js:40:18)
       │              at Compilation.reportDependencyErrorsAndWarnings (/Users/dchanp/Documents/amazon/codebase/OpenSearch-Dashboards/node_modules/webpack/lib/Compilation.js:1454:24)
       │              at /Users/dchanp/Documents/amazon/codebase/OpenSearch-Dashboards/node_modules/webpack/lib/Compilation.js:1258:10
       │              at _next0 (eval at create (/Users/dchanp/Documents/amazon/codebase/OpenSearch-Dashboards/node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:16:1)
       │              at eval (eval at create (/Users/dchanp/Documents/amazon/codebase/OpenSearch-Dashboards/node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:26:1)
       │              at processTicksAndRejections (node:internal/process/task_queues:95:5)
       │           @ ./node_modules/@opensearch-project/oui/lib/components/icon/index.js
       │           @ ./node_modules/@opensearch-project/oui/lib/components/index.js
       │           @ ./node_modules/@opensearch-project/oui/lib/index.js
       │           @ ./public/apps/configuration/unknown-datasource.tsx
       │           @ ./public/apps/configuration/panels/audit-logging/audit-logging.tsx
       │           @ ./public/apps/configuration/app-router.tsx
       │           @ ./public/apps/configuration/configuration-app.tsx
       │           @ ./public/plugin.ts
       │           @ ./public/index.ts
       │           @ /Users/dchanp/Documents/amazon/codebase/OpenSearch-Dashboards/packages/osd-optimizer/target/worker/entry_point_creator.js
```
</details>

Reply:
Removed the dependency due to compile time failures.